### DISCUSSION TITLE
Is Crankshaft currently compatible with RBPI5 ?

### DISCUSSION THREAD

**Discussion by sebdotffs** [opencardev/crankshaft#669] [General]
As the title says, I'm looking to make this for my car and I'm not sure if I should go with a new Pi5 or an older 4 board.

Thanks!

**Comment by Geekyadz** [comment]
Personally? I'd say get a pi 3 esp if you're looking to add a reversing camera.

However Pi 4 works if you want to not worry about using a reversing camera
